src/controllers/hero-list.js

```diff
@@ -1,4 +1,5 @@
 const heroService = require('../services/herolist');
+const { addHero } = require('../services/herolist');
 
 const getHeroes = (req, res) => {
   heroService.getHeroes(req.headers.userid)

```

Comment:
it must be better as "userId" to follow the style guide

Reply:
req.headers.userid comes directly from the header, also that's someone else's function that was already merged into master.